order-management-service/api/src/main/java/com/microservices/ordermanagement/api/dto/OrderStatus.java

```diff
@@ -3,8 +3,7 @@
 public enum OrderStatus {
     PENDING,
     CREATED,
-    APPROVED,
```

Comment:
no "approving" stage, huh? what about order accounting staff?

Reply:
from pending state you can wither get to created or to failed state